common/src/main/protobuf/network.proto

```diff
@@ -63,5 +63,6 @@ service Network {
   message FileReply {
     ResultType result = 1;
     string message = 2;
+    string file = 3;
```

Comment:
Does (string) file used in worker.scala?

Reply:
```string file``` is just a renamed variable which was message. Current ```string message``` is a literally message given to receiver(=worker) as a just reply message.